tests/unit/parsers/test.manifestjson.js

```diff
@@ -4335,13 +4398,17 @@ describe('ManifestJSONParser', () => {
         ],
       };
 
-      _checkKeySupport({ support, minVersion: 40 });
+      _checkKeySupport({
+        support,
+        minFirefoxVersion: 42,
+        minAndroidVersion: 54,
+      });
```

Comment:
what's the reason for editing this test vs. adding a new one?

Reply:
`checkKeySupport` is now called with both Firefox and Firefox for Android versions separately so I had to fix those tests. I edited the version numbers to make them more realistic too.